pkg/controller/eventing/subscription/reconcile_test.go

```diff
@@ -464,6 +465,33 @@ var testCases = []controllertesting.TestCase{
 				},
 			},
 		},
+	}, {
+		Name: "new subscription to non-existent K8s Service: fails with no service found",
+		InitialState: []runtime.Object{
+			getNewSubscriptionToK8sService(),
+		},
+		WantResult: reconcile.Result{},
+		WantPresent: []runtime.Object{
+			getNewSubscriptionToK8sServiceWithUnknownConditions(),
+		},
+		WantErrMsg: "services \"testk8sservice\" not found",
```

Comment:
Do we want this to say 'services "Foo" not found'? We probably want the singular "service" if this write is specific to core Services, or the type name (singular) if it's generic.

Reply:
that message doesn't come from us.

Reply:
good point.